doc/source/tutorial/stats.rst

```diff
@@ -178,6 +162,39 @@ keyword argument, ``loc``, which is the first of a pair of keyword arguments
 taken by all continuous distributions.
 This brings us to the topic of the next subsection.
 
+Random number generation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Drawing random numbers relies on generators from `numpy.random` package.
+In the examples above, the specific stream of
+random numbers is not reproducible across runs. To achieve reproducibility,
+you can explicitly *seed* the random number generator. In NumPy, a generator
+is an instance of `numpy.random.Generator`. Here is the canonical way to create
+a generator:
+
+    >>> from numpy.random import default_rng
+    >>> rng = default_rng()
+
+And fixing the seed can be done like this:
+
+    >>> from numpy.random import SeedSequence
+    >>> seed = SeedSequence().entropy
```

Comment:
This example doesn't fix the seed. Rerunning the whole example gets fresh entropy every time. Instead, I would put in an explicit number literal, like the old example. However, to improve on the old example, use a quality 128-bit number and in the `.. warning::` below show how you can use `SeedSequence().entropy` to get such a number.

Reply:
Ok thanks. I was really how to deal with this one. I should have said here that you could get a number by running the code and then use it as you pleased.

Ideally I did not want to put any number so that users would not copy past the same number all the time. I also thought about generating a fix number which would change every time you reload the page.